.yarn/patches/@credo-ts-anoncreds-npm-0.5.3-9fffe9ad63.patch

```diff
@@ -52,24 +52,39 @@ index 05f6bfef79ed716036430de6903348c11675a131..0f09bd59b8db5243e082a3f093d23bea
              });
          }
 diff --git a/build/updates/0.4-0.5/anonCredsCredentialRecord.js b/build/updates/0.4-0.5/anonCredsCredentialRecord.js
-index 915dffae9bd592aa489ef8ce5c31c9d3a8ce2ec3..b34aadfa008c6e5035823efbf139a044a8cabb24 100644
+index 915dffae9bd592aa489ef8ce5c31c9d3a8ce2ec3..559ae3352106e063ec7da045054bed096148fb2f 100644
 --- a/build/updates/0.4-0.5/anonCredsCredentialRecord.js
 +++ b/build/updates/0.4-0.5/anonCredsCredentialRecord.js
-@@ -112,6 +112,14 @@ async function migrateLegacyToW3cCredential(agentContext, legacyRecord) {
+@@ -112,6 +112,16 @@ async function migrateLegacyToW3cCredential(agentContext, legacyRecord) {
                  credentialRecordType: 'w3c',
                  credentialRecordId: w3cCredentialRecord.id,
              };
-+            // If using unqualified dids, store both qualified/unqualified revRegId forms
++            // If using Indy dids, store both qualified/unqualified revRegId forms
 +            // to allow retrieving it from revocation notification service
 +            if (legacyTags.revocationRegistryId && indyNamespace) {
++                const { credentialDefinitionTag, namespaceIdentifier, revocationRegistryTag, schemaSeqNo } =
++                    (0, indyIdentifiers_1.parseIndyRevocationRegistryId)(legacyTags.revocationRegistryId)
 +                relatedCredentialExchangeRecord.setTags({
 +                    anonCredsRevocationRegistryId: (0, indyIdentifiers_1.getQualifiedDidIndyDid)(legacyTags.revocationRegistryId, indyNamespace),
-+                    anonCredsUnqualifiedRevocationRegistryId: (0, indyIdentifiers_1.getUnQualifiedDidIndyDid)(legacyTags.revocationRegistryId),
++                    anonCredsUnqualifiedRevocationRegistryId: (0, indyIdentifiers_1.getUnqualifiedRevocationRegistryDefinitionId)(namespaceIdentifier, schemaSeqNo, credentialDefinitionTag, revocationRegistryTag),
```

Comment:
Should this fix be added to Credo or anoncrets-rs so that we can eventually not relay on the patch?

Reply:
It's partly in a PR and partly merged into Credo main after 0.5.3, so will likely make it into Credo 0.5.4 or 0.5.5. I didn't want to wait until then though